src/components/index.d.ts

```diff
@@ -1235,6 +1235,7 @@ export declare const NumberInput: {
     placeholderChar?: string
     guide?: boolean
     keepCharPositions?: boolean
+    maxLength?: number
```

Comment:
Do we also need to add this line to `export declare const TextMaskedInput: {...` further up in this file?